packages/nns/src/canisters/governance/response.converters.ts

```diff
@@ -680,6 +685,19 @@ const convertPbPrincipalIdToPrincipalString = (
 ): string =>
   Principal.fromUint8Array(pbPrincipal.getSerializedId_asU8()).toText();
 
+const convertNeuronSubaccountToAccountIdentifier = (
+  neuron: PbNeuron
+): string => {
+  const subAccount = SubAccount.fromBytes(
+    neuron.getAccount_asU8()
+  ) as SubAccount;
+
+  return AccountIdentifier.fromPrincipal({
+    principal: GOVERNANCE_CANISTER_ID,
+    subAccount: subAccount,
+  }).toHex();
```

Comment:
What do you think about returning the`AccountIdentifier` and do `toHex()` by the consumer of the function?

Reply:
good point. done